tests/rados/test_inconsistency_osd_epoch.py

```diff
@@ -50,27 +50,39 @@ def run(ceph_cluster, **kw):
         obj_list = rados_obj.get_object_list(pool_name)
         log.debug(f"All the objects added onto the pool are : {obj_list}")
 
-        oname = random.choice(obj_list)
+        oname_list = random.choices(obj_list, k=10)
```

Comment:
Changes here look good. Can we add the same workflow to other tests where we expect inconsistent to be present? like : `tests/rados/test_osd_inconsistency_pg.py`?

That can be as part of another PR as well if needed.

Reply:
I will take in another PR.